jubilant/statustypes.py

```diff
@@ -1,4 +1,4 @@
-"""Dataclasses used to hold parsed output from "juju status --format=json"."""
+"""Dataclasses used to hold parsed output from ``juju status --format=json``."""
```

Comment:
Minor, to avoid a misreading of "used to" as in "previously":

```suggestion
"""Dataclasses that contain parsed output from ``juju status --format=json``."""
```